sdccc/src/test/java/com/draeger/medical/sdccc/manipulation/precondition/impl/ManipulationPreconditionsTest.java

```diff
@@ -1455,162 +1282,122 @@ private static Stream<Arguments> metricStatusManipulationXActivationStateXArgume
                                 ManipulationPreconditions.MetricStatusManipulationSETActivationStateSHTDN::manipulation,
                         MetricCategory.SET,
-                        ComponentActivation.ON,
                         ComponentActivation.SHTDN),
                 // arguments for MetricStatusManipulationXActivationStateOFF preconditions
                 Arguments.of(
                         (Function<Injector, Boolean>)
                                 ManipulationPreconditions.MetricStatusManipulationMSRMTActivationStateOFF::manipulation,
                         MetricCategory.MSRMT,
-                        ComponentActivation.ON,
                         ComponentActivation.OFF),
                 Arguments.of(
                         (Function<Injector, Boolean>)
                                 ManipulationPreconditions.MetricStatusManipulationCLCActivationStateOFF::manipulation,
                         MetricCategory.CLC,
-                        ComponentActivation.ON,
                         ComponentActivation.OFF),
                 Arguments.of(
                         (Function<Injector, Boolean>)
                                 ManipulationPreconditions.MetricStatusManipulationSETActivationStateOFF::manipulation,
                         MetricCategory.SET,
-                        ComponentActivation.ON,
                         ComponentActivation.OFF),
                 // arguments for MetricStatusManipulationXActivationStateFAIL preconditions
                 Arguments.of(
                         (Function<Injector, Boolean>)
                                 ManipulationPreconditions.MetricStatusManipulationMSRMTActivationStateFAIL
                                         ::manipulation,
                         MetricCategory.MSRMT,
-                        ComponentActivation.ON,
                         ComponentActivation.FAIL),
                 Arguments.of(
                         (Function<Injector, Boolean>)
                                 ManipulationPreconditions.MetricStatusManipulationCLCActivationStateFAIL::manipulation,
                         MetricCategory.CLC,
-                        ComponentActivation.ON,
                         ComponentActivation.FAIL),
                 Arguments.of(
                         (Function<Injector, Boolean>)
                                 ManipulationPreconditions.MetricStatusManipulationSETActivationStateFAIL::manipulation,
                         MetricCategory.SET,
-                        ComponentActivation.ON,
                         ComponentActivation.FAIL));
     }
 
-    @DisplayName("The precondition is successful when setComponentActivation and setMetricStatus work as intended")
+    @DisplayName("The precondition is successful when setMetricStatus work as intended")
     @ParameterizedTest
     @MethodSource("metricStatusManipulationXActivationStateXArguments")
     void testMetricStatusManipulationXActivationStateXGood(
```

Comment:
Proposal: rename this method to "testMetricStatusManipulationXActivationStateYGood", because the type of Metric and the activation State are usually different...

Reply:
Good idea, i renamed this and other methods with similar naming pattern.